react/ProductList.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { FC } from 'react'
+import { Product } from 'vtex.catalog-graphql'
```

Comment:
I didn't know this existed 🤔

Reply:
Yes! You can run `vtex setup --typings` to import the types from the schema of the dependencies